test/e2e/app-dir/dynamic-io-errors/fixtures/sync-attribution/unguarded-async-unguarded-clientsync/app/page.tsx

```diff
@@ -0,0 +1,33 @@
+import { cookies } from 'next/headers'
+
+import { SyncIO } from './client'
+
+export default async function Page() {
+  return (
+    <main>
+      <section>
+        In This test we have a Client Component that accesses sync IO in a
+        Suspense boundary and a Server Component that accesses Request data
+        without a Suspense Boundary. We expect this to error because of the
+        Request data access and need to assert that the error does not
+        incorrectly indicate that sync IO is the problem.
```

Comment:
The client component also does not have a Suspense boundary. And the test asserts that sync IO is indeed the problem in this scenario.